docs/source/release/v6.0.0/mantidworkbench.rst

```diff
@@ -15,6 +15,7 @@ New Features
 - Added the ability to load 1d-Plots from MantidPlot project files in workbench.
 - Added QLineEdit to SampleLogs widget to allow filtering its entries
 - Time series sample logs in multiperiod event workspaces can now be filtered by current period.
+- Added a setting to control the positional (ontop vs floating) behavior of plot windows.
```

Comment:
With the interfaces now being effected by this pr the release note should be updated.